components/faq/questions.js

```diff
@@ -222,7 +222,8 @@ const questions = {
     },
     {
       question:
-        "I'd like to donate my software conference pass to an Operation Code member. How do I do that?",
+        "I'd like to donate my software conference pass" +
```

Comment:
To avoid a string concatenation + avoid an eslint error, you could just do the entire string in backticks `

Reply:
ok will do! thank you